Makefile

```diff
@@ -65,6 +79,43 @@ test-tvos:
 
+unit-test-ios:
+	@echo "######################################################################"
+	@echo "### Unit Testing iOS"
+	@echo "######################################################################"
+	xcodebuild test -workspace $(PROJECT_NAME).xcworkspace -scheme $(UNIT_TEST_SCHEME) -destination 'platform=iOS Simulator,name=iPhone 8' -derivedDataPath build/outn -resultBundlePath iosunittestresults.xcresult -enableCodeCoverage YES
+
+unit-test-tvos:
+	@echo "######################################################################"
+	@echo "### Unit Testing tvOS"
+	@echo "######################################################################"
+	xcodebuild test -workspace $(PROJECT_NAME).xcworkspace -scheme $(UNIT_TEST_SCHEME) -destination 'platform=tvOS Simulator,name=Apple TV' -derivedDataPath build/outn -resultBundlePath tvosunittestresults.xcresult -enableCodeCoverage YES
+
+functional-test-ios:
+	@echo "######################################################################"
+	@echo "### Functional Testing iOS"
+	@echo "######################################################################"
+	xcodebuild test -workspace $(PROJECT_NAME).xcworkspace -scheme $(FUNCTIONAL_TEST_SCHEME) -destination 'platform=iOS Simulator,name=iPhone 8' -derivedDataPath build/outn -resultBundlePath iosfunctionaltestresults.xcresult -enableCodeCoverage YES
+
+functional-test-tvos:
+	@echo "######################################################################"
+	@echo "### Functional Testing tvOS"
+	@echo "######################################################################"
+	xcodebuild test -workspace $(PROJECT_NAME).xcworkspace -scheme $(FUNCTIONAL_TEST_SCHEME) -destination 'platform=tvOS Simulator,name=Apple TV' -derivedDataPath build/outn -resultBundlePath tvosfunctionaltestresults.xcresult  -enableCodeCoverage YES
+
+integration-test-ios:
+	@echo "######################################################################"
+	@echo "### Integration Testing iOS"
+	@echo "######################################################################"
+	xcodebuild test -workspace $(PROJECT_NAME).xcworkspace -scheme $(INTEGRATION_TEST_SCHEME) -destination 'platform=iOS Simulator,name=iPhone 8' -derivedDataPath build/outn -resultBundlePath iosintegrationtestresults.xcresult -enableCodeCoverage YES
+
+integration-test-tvos:
+	@echo "######################################################################"
+	@echo "### Integration Testing tvOS"
+	@echo "######################################################################"
+	xcodebuild test -workspace $(PROJECT_NAME).xcworkspace -scheme $(INTEGRATION_TEST_SCHEME) -destination 'platform=tvOS Simulator,name=Apple TV' -derivedDataPath build/outn -resultBundlePath tvosintegrationtestresults.xcresult -enableCodeCoverage YES
+
+
```

Comment:
You are not referencing these individual test tasks in the CI configuration. I'd say to remove these if they are not used in CI as developers can run these directly from Xcode.